__tests__/app.test.js

```diff
@@ -129,3 +132,66 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("POST - should add a comment for an article", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: "this is a test comment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        expect(response.body.comment).toBeInstanceOf(Object);
+        expect(response.body.comment).toEqual({
+          article_id: 1,
+          comment_id: 19,
+          votes: 0,
+          created_at: expect.any(String),
+          author: "icellusedkars",
+          body: "this is a test comment",
+        });
+      });
+  });
+
+  test("POST - status should be 400 when username does not exist", () => {
+    const newComment = {
+      username: "emran",
+      body: "this is a test comment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(400)
```

Comment:
I think this hsould change to a 404 - tefchnically the username emran _could_ exist, but it just doesn't yet, so it's realistically a 404 not found :)